--- conf/aero.conf	2016-05-18 15:24:52.000000000 +0300
+++ conf_orig/aero.conf	2017-04-27 09:39:29.324590000 +0300
@@ -1,60 +1,60 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<config>
-   <request_config>false</request_config>
-   <config_request_address>ws://localhost:2347</config_request_address>
-
-   <request_earth>false</request_earth>
-   <earth_file>conf/aero.earth</earth_file>
-   <earth_request_address>ws://localhost:2348</earth_request_address>
-
-   <request_flight_info_com>false</request_flight_info_com>
-   <com_port>COM3</com_port>
-
-   <request_flight_info_web>true</request_flight_info_web>
-   <web_address>ws://192.168.0.162:2345</web_address>
-   <request_flight_info_interval>10</request_flight_info_interval>
-
-   <request_flight_archive>false</request_flight_archive>
-   <flight_archive_request_address>ws://192.168.0.162:2346</flight_archive_request_address>
-   <fixed_flight_height>80000</fixed_flight_height>
-
-   <model_path>model_il96.obj.50.scale.0,0,180.rot.(0,-10000,0).trans</model_path>
-   <log_path>aero.log</log_path>
-
-   <display_mode>auto</display_mode>
-   <manual_mode_timeout>30</manual_mode_timeout>
-   <min_camera_altitude_manual>50500</min_camera_altitude_manual>
-   <max_camera_altitude_manual>20100500</max_camera_altitude_manual>
-
-   <language>ru</language>
-
-   <show_flight_infowindow>false</show_flight_infowindow>
-   <flight_infowindow_interval>15</flight_infowindow_interval>
-   <flight_infowindow_showtime>5</flight_infowindow_showtime>
-
-   <show_object_infowindow>false</show_object_infowindow>
-   <object_infowindow_interval>20</object_infowindow_interval>
-   <object_infowindow_showtime>5</object_infowindow_showtime>
-   <infowindow_header_font>../arial.ttf</infowindow_header_font>
-   <infowindow_header_font_size>30</infowindow_header_font_size>
-   <infowindow_body_font>../arial.ttf</infowindow_body_font>
-   <infowindow_body_font_size>20</infowindow_body_font_size>
-  
-   <objects_features_name>cities_labels</objects_features_name>
-   <objects_photo_dir>../../../geodata/photo</objects_photo_dir>  <!-- this folder is abs or relative to AerogisWidget.qml, must contain photos -->
-   <objects_text_dir>../../geodata/text</objects_text_dir>  <!-- this folder is abs or relative to .exe, must contain en ru es dirs -->
-   <max_object_distance>1000000</max_object_distance>
-
-   <trace_style></trace_style>
-   <map_labels_font>Arial</map_labels_font>
-   <map_labels_font_size>14</map_labels_font_size>
-   <infowindow_bgcolor>#80202020</infowindow_bgcolor>
-   <infowindow_textcolor>#ffffffff</infowindow_textcolor>
-
-   <save_images>true</save_images>
-   <save_images_gui>true</save_images_gui>
-   <save_images_interval>10</save_images_interval>
-   <save_images_stack_size>20000000</save_images_stack_size>
-   <save_images_path>screens</save_images_path>
-
-</config>
+<?xml version="1.0" encoding="UTF-8"?>
+<config>
+   <request_config>false</request_config>
+   <config_request_address>ws://localhost:2347</config_request_address>
+
+   <request_earth>false</request_earth>
+   <earth_file>conf/aero.earth</earth_file>
+   <earth_request_address>ws://localhost:2348</earth_request_address>
+
+   <request_flight_info_com>true</request_flight_info_com>
+   <com_port>COM2</com_port>
+   <request_flight_info_web>false</request_flight_info_web>
+   <web_address>ws://localhost:2345</web_address>
+   <request_flight_info_interval>20</request_flight_info_interval>
+
+   <request_flight_archive>false</request_flight_archive>
+   <flight_archive_request_address>ws://localhost:2346</flight_archive_request_address>
+   <fixed_flight_height>80000</fixed_flight_height>
+
+   <model_path>../../geodata/models/model_il96.obj.50.scale.0,0,180.rot.(0,-10000,0).trans</model_path>
+   <trace_color>#2a7aab50</trace_color>
+   
+   <log_path>../log/aero.log</log_path>
+
+   <display_mode>auto</display_mode>
+   <manual_mode_timeout>30</manual_mode_timeout>
+   <min_camera_altitude_manual>50500</min_camera_altitude_manual>
+   <max_camera_altitude_manual>20100500</max_camera_altitude_manual>
+
+   <language>en</language>
+
+   <show_flight_infowindow>false</show_flight_infowindow>
+   <flight_infowindow_interval>15</flight_infowindow_interval>
+   <flight_infowindow_showtime>5</flight_infowindow_showtime>
+
+   <show_object_infowindow>true</show_object_infowindow>
+   <object_infowindow_interval>30</object_infowindow_interval>
+   <object_infowindow_showtime>10</object_infowindow_showtime>
+   <infowindow_header_font>../arial.ttf</infowindow_header_font>
+   <infowindow_header_font_size>20</infowindow_header_font_size>
+   <infowindow_body_font>../arial.ttf</infowindow_body_font>
+   <infowindow_body_font_size>14</infowindow_body_font_size>
+  
+   <objects_features_name>cities_labels</objects_features_name>
+   <objects_photo_dir>../../../geodata/photo</objects_photo_dir>  <!-- this folder is abs or relative to AerogisWidget.qml, must contain photos -->
+   <objects_text_dir>../../geodata/text</objects_text_dir>  <!-- this folder is abs or relative to .exe, must contain en ru es dirs -->
+   <max_object_distance>1000000</max_object_distance>
+
+   <infowindow_textcolor>#ffffffff</infowindow_textcolor>
+   <infowindow_bgcolor>#20202080</infowindow_bgcolor>
+
+   <save_images_3lang>true</save_images_3lang>
+   <save_images_gui>false</save_images_gui>
+   <save_images_interval>0</save_images_interval>
+   <save_images_when_new_point>true</save_images_when_new_point>
+   <save_images_stack_size>20000000</save_images_stack_size>
+   <save_images_path>screens</save_images_path>
+   <save_images_top low="500000" med="2000000" hi="3600000">true</save_images_top>
+
+</config>
